cn.lceda.LCEDAPro.yml

```diff
@@ -0,0 +1,60 @@
+app-id: cn.lceda.LCEDAPro
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+command: lceda-pro.sh
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it really support wayland out of the box? if yes then line 13 should be fallback-x11 otherwise, line 14 should be removed.